ros2_batch_job/__main__.py

```diff
@@ -356,6 +371,51 @@ def build_and_test(args, job):
     # return 0 if ret_test == 0 and ret_testr == 0 else 1
     return 0
 
+def build_with_sanitizer(cmd, args, job):
+    print('# BEGIN SUBSECTION: build with sanitizer')
+    sanitizer_type = args.enable_sanitizer_type
+    sanitizer_cmake_args = [
+    '-DCMAKE_CXX_FLAGS="-fsanitize=' + sanitizer_type + ' -g"',
```

Comment:
Rule of thumb is if you end up having more than one + in a string expression, format is probably what you should use instead: https://www.geeksforgeeks.org/python-format-function/

"-DCMAKE_CXX_FLAGS='{} -g'".format(sanitizer_type)

It also helps you declaring those strings as top-level constants.

Reply:
Makes sense, done